templates/request/get/get.handlebars

```diff
@@ -0,0 +1,19 @@
+    it('should be {{description}}', function({{#if asynchronous}}done{{/if}}) {
+      request({
+        url: '{{url}}',
+        method: 'GET',
+        headers: {'Content-Type': 'application/json'}
+      },
+      function(error, response, body) {
+        {{#compare assertion 'expect'}}
+        chai.expect(body).to.have.property('name');
```

Comment:
Lets add some comments here about the generated code. It should be clear that it's dummy code